image/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM opensuse/leap:15.5
-MAINTAINER Julio Gonzalez Gil <jgonzalez@suse.com>
```

Comment:
Any reason we're removing the Maintainer? Feel free to change it so you are, but we should leave one :-)